include/deal.II/base/data_out_base.h

```diff
@@ -1035,7 +1035,7 @@ namespace DataOutBase
      * this number is used only to verify that the format we are writing is
      * what the current readers and writers understand.
      */
-    static const unsigned int format_version = 3;
+    static unsigned int format_version;
```

Comment:
I think you want to put the `const` back in here, and in the definition. Or is there a good reason to remove it?